src/overview/build-deploy.md

```diff
@@ -9,36 +9,36 @@ Every time you push to a live branch (a git branch with an active environment at
+After you push your code, the first build step is to validate your configuration files (i.e. `.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml`). The Git server will issue an error if validation fails, and nothing will happen on the server.
 
-> While most projects have a single `.platform.app.yaml` file, Platform.sh supports multiple applications in a single project.  It will scan the repository for `.platform.app.yaml` files in subdirectories and will build each subdirectory that contains one as an independent application. The built application will not contain any directories above the one in which it is found. The system is smart enough not to rebuild applications that have already been built, so if you have multiple applications, only the one that changed will be rebuilt and redeployed.
+> While most projects have a single `.platform.app.yaml` file, Platform.sh supports multiple applications in a single project.  It will scan the repository for `.platform.app.yaml` files in subdirectories and each directory containing one will be built as an independent application. A built application will not contain any directories above the one in which it is found. The system is smart enough not to rebuild applications that have already been built, so if you have multiple applications, only changed applications will be rebuilt and redeployed.
 
-The live environment is composed of multiple containers - both for your application(s) and for  the services it depends on. It also has a virtual network that connects them as well as a router that routes incoming requests to you application(s).
+The live environment is composed of multiple containers&mbdash;both for your application(s) and for the services it depends on. It also has a virtual network connecting them, as well as a router to route incoming requests to the appropriate application.
 
 Based on your application type the system will select one of our pre-built container images and run the following:
 
-1. First, any dependencies specified in the `.platform.app.yaml` file are installed.  Those include tools like Sass, Gulp, Drupal Console, or many others that you may need.  
+1. First, any dependencies specified in the `.platform.app.yaml` file are installed.  Those include tools like Sass, Gulp, Drupal Console, or any others that you may need.  
 
-2. Second, depending on the “build flavor” specified in the configuration file we run a series of standard commands. The default for PHP containers, for example, is simply to run `composer install`.
+2. Then, depending on the “build flavor” specified in the configuration file, we run a series of standard commands. The default for PHP containers, for example, is simply to run `composer install`.
```

Comment:
I disagree with both changes on this line.  First should be followed by Second, and the comma is extraneous.

Reply:
This is pretty much a preference thing, but on a list of three, I think it's stylistically better to use "First,...Then,...Finally." with "second" reserves for lists of four or more.

Reply:
Also, introductory words, phrases, and clauses get a comma. If you can lop off the words at the beginning of a sentence and it's not an descriptor of the word at the beginning of the sentence, it almost always requires a comma.

Reply:
I was referring to the comma after "configuration file".  I agree that the Second/Then should have a comma.

Reply:
"depending on the 'build flavor' specified in the configuration file" is a subordinate clause. Comma!